.github/workflows/nox.yml

```diff
@@ -19,44 +19,64 @@ env:
   FORCE_COLOR: "1"
 
 jobs:
-  nox-lint:
+  nox:
     runs-on: ubuntu-latest
     defaults:
       run:
         working-directory: antsibull
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - session: test
+            python-versions: "3.9, 3.10, 3.11"
+            codecov: true
+            packages: ""
+
+          - session: "check_package_files(8.1.0)"
+            python-versions: "3.11"
+            codecov: true
+            packages: ""
+
+          - session: "check_package_files(7.5.0)"
+            python-versions: "3.11"
+            codecov: true
+            packages: ""
+
+          - session: "check_package_files(8.1.0_setup_cfg)"
+            python-versions: "3.11"
+            codecov: true
+            packages: ""
+
+          - session: lint
+            python-versions: "3.11"
+            codecov: false
+            packages: ""
+    name: "Run nox ${{ matrix.session }} session"
     steps:
       - name: Check out antsibull
         uses: actions/checkout@v3
         with:
           path: antsibull
-      - name: Setup nox
-        uses: wntrblm/nox@2023.04.22
-        with:
-          python-versions: "3.11"
-      - name: nox -e lint
+      - name: Install extra packages
         run: |
-          OTHER_ANTSIBULL_MODE=git nox -v -e lint
-  nox-test:
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: antsibull
-    steps:
-      - name: Check out antsibull
-        uses: actions/checkout@v3
-        with:
-          path: antsibull
+          if [ -n "${{ matrix.packages }}" ]; then
```

Comment:
Why not skip this step if the string is empty? Tat's more efficient as GHA doesn't need to run something.

Reply:
Can you remind me of the syntax for that?

Reply:
`if: matrix.packages != ""` should work, as in https://github.com/ansible-community/github-docs-build/blob/main/.github/workflows/_shared-docs-build-pr.yml#L370

Reply:
Let's see if https://github.com/ansible-community/antsibull/pull/530/commits/6ea86877cceb9b8e3b2963e7411ea1baa8ea2fc6 works...

Reply:
To really see it we would actually need an example where `packages` is not empty :)